wpdetect/__main__.py

```diff
@@ -1,154 +1,111 @@
-import urllib.request
+import click
 import sys
-from pyfiglet import figlet_format
-
-wp_domains = []
-header = "'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.75.14 (KHTML, like Gecko) Version/7.0.3 Safari/7046A194A'"
-def wp_check(url):
-    url_wpl = url + "/wp-login.php"
-    url_wpac = url + "/wp-content/"
-    url_wpad = url + "/wp-admin/"
-    url_wpc = url + "/wp-cron.php"
-    url_wpx = url + "/xmlrpc.php"
-    url_wpa = url + "/wp-json/wp/v2/"
-    url_wpact = url + "/wp-content/themes/"
-
-    req_wpl = urllib.request.Request(url_wpl, headers={'User-Agent': header})
-    req_wpac = urllib.request.Request(url_wpac, headers={'User-Agent': header})
-    req_wpact = urllib.request.Request(url_wpact, headers={'User-Agent': header})
-    req_wpacp = urllib.request.Request(url_wpact, headers={'User-Agent': header})
-    req_wpad = urllib.request.Request(url_wpad, headers={'User-Agent': header})
-    req_wpc = urllib.request.Request(url_wpc, headers={'User-Agent': header})
-    req_wpx = urllib.request.Request(url_wpx, headers={'User-Agent': header})
-    req_wpa = urllib.request.Request(url_wpa, headers={'User-Agent': header})
-
-    print("Please wait.../")
-    try:
```

Comment:
I would request you improve this long list of if-else ladder maybe use a dictionary or something to do this in a better way.

Reply:
Right now the default method is the fast one. The old code will be implemented as the slow method